src/App.tsx

```diff
@@ -1,27 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goodsFromServer, setgoodsFromServer] = useState<Good[]>([]);
+  const loadGoods = async () => {
+    const goods = await getAll();
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+    setgoodsFromServer(goods);
+  };
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+  const loadFiveGoods = async () => {
+    const goods = await get5First();
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+    setgoodsFromServer(goods);
+  };
 
-    <GoodsList goods={[]} />
-  </div>
-);
+  const loadOnlyRedGoods = async () => {
+    const goods = await getRedGoods();
+
+    setgoodsFromServer(goods);
+  };
+
+  return (
+    <div className="App">
+      <h1>Dynamic list of Goods</h1>
+
+      <button type="button" data-cy="all-button" onClick={loadGoods}>
```

Comment:
will be better to read each atr on new str

Reply:
agree, fix @Yevheniya-Deryka comment